modules/twinklexfd.js

```diff
@@ -1616,6 +1703,28 @@ Twinkle.xfd.callback.evaluate = function(e) {
 			// find target and pass main as the callback
 			Twinkle.xfd.callbacks.rfd.findTarget(params, Twinkle.xfd.callbacks.rfd.main);
 			break;
+
+		case 'rm':
+			params = { reason: reason, newname: newname, rmtr: rmtr };
+			var talkpagename = mw.Title.newFromText(Morebits.pageNameNorm).getTalkPage().getPrefixedText();
+			Morebits.wiki.actionCompleted.redirect = rmtr ? 'Wikipedia:Requested moves/Technical requests' : talkpagename;
```

Comment:
regarding redundancy reduction - done. 

Originally intended for parallelism, yeah. But I now see that it's possible to use .append() without really harming parallelism, instead of `wikipedia_page.load(callback)`, by just doing `callback(wikipedia_page)` ! So changed.

Reply:
Jah.  Mostly just used for notifications to creators, but that's basically what this is (just like talkpage notice of merge nom).